tests/common.py

```diff
@@ -3,19 +3,30 @@
 from pywps.tests import WpsClient, WpsTestResponse
 
 import os
+import re
 
 VERSION = "1.0.0"
 xpath_ns = get_xpath_ns(VERSION)
 TESTS_HOME = os.path.abspath(os.path.dirname(__file__))
 
+DATA_DIR = os.path.join(TESTS_HOME, "data")
```

Comment:
It is more robust to use `pkg_resource.resource_filename()` and `pkg_resource.resource_stream()` from Python's standard library. This ensures that the files are referenced relative to whether the package gets installed.

See [an example usage](https://github.com/pacificclimate/crmprtd/blob/master/tests/conftest.py#L512) in our `crmprtd` package and [how to set it up](https://github.com/pacificclimate/crmprtd/blob/master/setup.py#L53) in the package config.